src/scripts/main.js

```diff
@@ -1,3 +1,32 @@
 'use strict';
 
-// write code here
+function convertToNumber(stringSalary) {
+  return parseFloat(stringSalary.replace(/[^\d.-]/g, ''));
+}
+
+function sortList(list) {
+  const itemsList = [...list.children];
+
+  itemsList.sort((a, b) => {
+    const salaryA = convertToNumber(a.dataset.salary);
+    const salaryB = convertToNumber(b.dataset.salary);
+
+    return salaryB - salaryA;
+  });
+
+  list.innerHTML = '';
+
+  itemsList.forEach((item) => list.appendChild(item));
+}
+
+function getEmployees(list) {
+  const items = [...list.children];
+
+  return items.map((i) => ({
+    name: i.dataset.name,
+    position: i.dataset.position,
+    salary: convertToNumber(i.dataset.salary),
+  }));
```

Comment:
The `getEmployees` function is missing the `age` property in the employee objects. Ensure that the returned objects include `name`, `position`, `salary`, and `age` as specified in the task.